src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/SetAzureSqlFailoverGroup.cs

```diff
@@ -93,9 +97,11 @@ protected override IEnumerable<AzureSqlFailoverGroupModel> ApplyUserInputToModel
             AzureSqlFailoverGroupModel newModel = model.First();
 
             newModel.ReadWriteFailoverPolicy = MyInvocation.BoundParameters.ContainsKey("FailoverPolicy") ? FailoverPolicy.ToString() : newModel.ReadWriteFailoverPolicy;
+#pragma warning disable 0618
```

Comment:
@ziwa-msft pleaes revert this, all this will not be necessary when you make the breaking change, you will have to delete this later. So why introduce now when you don't even need it.